src/main/java/com/launchacademy/giantleap/models/User.java

```diff
@@ -32,6 +32,13 @@
 
   private String password;
 
+//  additional add to correlate to the  new schema.- Clyde
+   @Column(name="is_bar_owner")
+  private Boolean isBarOwner;
```

Comment:
please remove comments before submitting PRs
Also, watch indentation. @column should be backed up to line up with private